Voice Note/Views/RecordingListView.swift

```diff
@@ -10,21 +10,19 @@ import CoreLocation
 
 struct RecordingListView: View {
     @State var toast:ToastView? = nil
-    //@Environment(\.managedObjectContext) private var managedObjectContext
     @Environment(\.coreData) private var coreDataService: CoreDataService
```

Comment:
@Environment(\.managedObjectContext) private var managedObjectContext was there if we need it at some point